verification/verify_test.go

```diff
@@ -135,3 +150,59 @@ func TestVerify(t *testing.T) {
 	assert.Equal(err, verifier.err)
 	assert.Nil(res)
 }
+
+func TestPixels(t *testing.T) {
+	ffmpeg.InitFFmpeg()
+
+	assert := assert.New(t)
+
+	p, err := Pixels("foo")
+	assert.EqualError(err, "No such file or directory")
+	assert.Equal(int64(0), p)
+
+	// Assume that ffmpeg.Transcode3() returns the correct pixel count so we just
+	// check that no error is returned
+	p, err = Pixels("../server/test.flv")
+	assert.Nil(err)
+	assert.NotZero(p)
+}
+
+func TestVerifyPixels(t *testing.T) {
+	ffmpeg.InitFFmpeg()
+
+	require := require.New(t)
+	assert := assert.New(t)
+
+	// Create memory session and save a test file
+	bos := drivers.NewMemoryDriver(nil).NewSession("foo")
+	data, err := ioutil.ReadFile("../server/test.flv")
+	require.Nil(err)
+	fname, err := bos.SaveData("test.ts", data)
```

Comment:
Do we need the memory session at all here? Seems like we can just pass the `data` var initialized above directly into `verifyPixels`?